packages/cli/package.json

```diff
@@ -52,6 +52,7 @@
     "chromedriver": "latest",
     "colors": "^1.4.0",
     "commander": "^9.4.1",
+    "dotenv": "^16.4.5",
```

Comment:
Makes sense - this was missed out in https://github.com/dequelabs/axe-core-npm/commit/4cce5cbf7cc0ef3b93afbe92bbe6b82d8dc38a22 as a new dep